tests/test_data/test_utils.py

```diff
@@ -11,3 +13,18 @@ def connect_to_mem_db():
         'url': {'protocol': 'sqlite'},
     }
     return SqlAlchemyConnectionRegistry(OmegaConf.create(conf))
+
+
+host = 'server.example.com'

```

Comment:
if this variables are not being reused. put them directly into the conf dict returned from `connect_to_mongo`